vocabularies/Graph.xml

```diff
@@ -33,39 +36,27 @@
         </Collection>
       </Annotation>
 
-      <Term Name="links" Type="Collection(Graph.link)" Nullable="false" AppliesTo="EntityType">
+      <Term Name="traceId" Type="Edm.String" Nullable="false">
         <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.Description" String="Navigation links" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.Description" String="The traceId contains a unique string that is preserved across multiple requests and log files. It is used in error responses to help diagnose problems by correlating log entries." />
       </Term>
 
-      <ComplexType Name="link">
+      <Term Name="Details" Type="Graph.DetailsType" Nullable="false">
         <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.Description" String="Navigation link" />
-        <Property Name="title" Type="Edm.String">
-          <Annotation Term="Core.Description" String="Title of the navigation link" />
-        </Property>
-        <Property Name="alias" Type="Edm.String">
-          <Annotation Term="Core.Description" String="Name of the virtual navigation property" />
-        </Property>
-        <Property Name="target" Type="Edm.String">
-          <Annotation Term="Core.Description" String="Name of the target entity set" />
-        </Property>
-        <Property Name="parameters" Type="Collection(Graph.binding)" Nullable="false">
-          <Annotation Term="Core.Description" String="Map of parameter names in the target operation to property paths in the annotated entity type" />
-        </Property>
-        <Property Name="filter" Type="Collection(Graph.binding)" Nullable="false">
-          <Annotation Term="Core.Description" String="Map of single-valued property paths in the target entity to property paths in the annotated entity type" />
-          <Annotation Term="Core.LongDescription" String="Used to construct a `$filter` expression and triggers the `Read` (list) operation on the target entity set" />
-        </Property>
-      </ComplexType>
-
-      <ComplexType Name="binding">
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.Description" String="Graph-specific details for error responses" />
```

Comment:
If it is only for error responses, shouldn't the term be called `ErrorDetails`?

Reply:
Given that it is directly within the `error` object of an OData error response, that would look redundant. Now it is:
~~~jsonc
"error": {
  "@Graph.Details": {
     // ...
  }
}
~~~